superset/views/core.py

```diff
@@ -554,11 +554,6 @@ class DashboardModelView(SupersetModelView, DeleteMixin):  # noqa
             'want to alter specific parameters.'),
         'owners': _('Owners is a list of users who can alter the dashboard.'),
     }
-    base_filters = [['slice', DashboardFilter, lambda: []]]
```

Comment:
I think this is the way we make sure people see only the dashboards they have access to.